welcomescreen.py

```diff
@@ -44,14 +47,23 @@ def run(self):
         land = pygame.image.load("assets/land.png").convert()
         land1 = pygame.transform.scale(land, (490, 150))
         land2 = land1
-        sky = pygame.image.load("assets/sky.png").convert()
-        sky = pygame.transform.scale(sky, (490, 200))
+        background = [
+            (i, i) for i in (
+                pygame.transform.scale(
+                    pygame.image.load("assets/sky3.png").convert(), (490, 33 * (200 / 109)),
+                ),
+                pygame.transform.scale(
+                    pygame.image.load("assets/sky2.png").convert_alpha(), (490, 40 * (200 / 109)),
+                ),
+                pygame.transform.scale(
+                    pygame.image.load("assets/sky1.png").convert_alpha(), (490, 44 * (200 / 109)),
+                ),
+            )
+        ]
```

Comment:
This is quite difficult to read, could you make it better? Since the only thing that changes are the images then maybe a function that does that would be helpful.